clustered/client/src/main/java/org/ehcache/clustered/client/internal/service/DefaultClusteringService.java

```diff
@@ -90,6 +91,7 @@
   DefaultClusteringService(final ClusteringServiceConfiguration configuration) {
     this.configuration = configuration;
     URI ehcacheUri = configuration.getClusterUri();
+    this.properties = configuration.getProperties();
```

Comment:
My problem here is that the timeout is coming from another place (that will override this). I don't like properties coming from two places. I don't have the best answer but I don't think it's a good idea.